src/libbson/fuzz/fuzz_test_validate.c

```diff
@@ -0,0 +1,18 @@
+#include <stdlib.h>
+#include <stdio.h>
+#include <stdint.h>
+#include <bson/bson.h>
+
+int
+LLVMFuzzerTestOneInput (const uint8_t *data, size_t size)
+{
+   bson_t b;
+   if (bson_init_static (&b, data, size)) {
+      bson_validate (&b,
+                     BSON_VALIDATE_NONE | BSON_VALIDATE_UTF8 | BSON_VALIDATE_DOLLAR_KEYS | BSON_VALIDATE_DOT_KEYS |
```

Comment:
```suggestion
                     BSON_VALIDATE_UTF8 | BSON_VALIDATE_DOLLAR_KEYS | BSON_VALIDATE_DOT_KEYS |
```

Suggest removing `BSON_VALIDATE_NONE` (evaluates to 0).